Gemfile

```diff
@@ -1,20 +1,17 @@
 source "https://rubygems.org"
 
-gem "sinatra"
-gem "sinatra-activerecord"
+gem 'actionmailer'
 gem 'pg'
+gem 'sinatra'
+gem 'sinatra-activerecord'
```

Comment:
Use either single or double quotes, it really doesn't matter - but be consistent.

Some Rubyists would say always double quotes unless you *need* single quotes, others would say the opposite - default to double quotes... Honestly who cares, as long as you can explain your choice - and as long as you're consistent (that's something people *will* pick up on when reviewing code)